### PR TITLE
Web refact

### PR DESCRIPTION
**Pull request by msarmi9**
This PR adds no new features. It simply refactors `classes.py` into 3 new modules:

* `forms.py` -- contains `UserForm`, `PatientForm`, etc.
* `persons.py` -- contains `User`, `Patient`
* `medication.py` -- contains `Prescription`, `Intake`, etc.

Walked through the basic functions of the web app manually and all seems to work still (we need to get some automated tests up to do this for us).

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+class RegistrationForm(InfoForm, UserForm):
+    """Final class for registering new users (doctors)."""
+
+    submit = SubmitField("Submit")
+
+
+class LoginForm(UserForm):
+    """Final class for logging in existing users (doctors)."""
+
+    subimt = SubmitField("Login")
```
Comment (ajcheon) [inline app/forms.py]: Typo `submit`

File: app/routes.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 from app import application, db
 from app.utils import *
-from app.classes import (
-    User,
-    Patient,
-    Prescription,
-    PrescriptionForm,
-    Patient,
+from app.medication import Prescription, Intake
+
```
Comment (ajcheon) [inline app/routes.py]: Don't need the spaces here

**Review by ajcheon — COMMENTED** [review]
Nice work, @msarmi9 thanks!  Just a typo in the submit field for a form, not super important now but probably better fixed in case we ever need to use it and are wondering why `submit` is not working.

File: app/forms.py (hunk 1 above)
Comment (msarmi9) [reply app/forms.py]: lol thnx for this! i must not have rebuilt the image when i tested it, because otherwise it wouldn't have worked

**Review by ajcheon — APPROVED** [review]

**Pull request merged**